language/en_UK.json

```diff
@@ -28,15 +28,15 @@
         },
         "webhookurl": {
             "generate_key_first": "You need to generate an authorization token first using `/settings set authkey`.",
-            "success": "The url is: `{url}`.\nPlease use this url as the discord webhook url for the suggestions module."
```

Comment:
Why is suggestions with a capital S?

Reply:
Because it is a module and not a normal word `suggestions`?